src/app/zones/views/ZoneIngressListView.vue

```diff
@@ -13,142 +13,144 @@
         },
       ]"
     >
-      <div class="zoneingresses">
-        <MultizoneInfo v-if="store.getters['config/getMulticlusterStatus'] === false" />
-
-        <!-- Zone CPs information for when Multicluster is enabled -->
-        <div
-          v-else
-          class="stack"
+      <template #title>
+        <h2>
+          <RouteTitle
+            :title="t('zone-ingresses.routes.items.title')"
+            :render="true"
+          />
+        </h2>
+      </template>
+
+      <MultizoneInfo v-if="store.getters['config/getMulticlusterStatus'] === false" />
+
+      <template v-else>
+        <DataSource
+          v-slot="{ data, error }: ZoneIngressOverviewCollectionSource"
+          :src="`/zone-ingresses?size=${props.size}&page=${props.page}`"
         >
-          <div class="kcard-border">
-            <DataOverview
-              :selected-entity-name="entity?.name"
-              :page-size="PAGE_SIZE_DEFAULT"
-              :is-loading="isLoading"
-              :error="error"
-              :empty-state="EMPTY_STATE"
-              :table-data="tableData"
-              :table-data-is-empty="tableData.data.length === 0"
-              :next="nextUrl"
-              :page-offset="pageOffset"
-              @table-action="loadEntity"
-              @load-data="loadData"
-            />
-          </div>
-
-          <div
-            v-if="entity !== null"
-            class="kcard-border"
-            data-testid="list-view-summary"
-          >
-            <ZoneIngressDetails :zone-ingress-overview="entity" />
-          </div>
-        </div>
-      </div>
+          <KCard>
+            <template #body>
+              <AppCollection
+                class="zone-ingress-collection"
+                data-testid="zone-ingress-collection"
+                :headers="[
+                  { label: 'Name', key: 'name' },
+                  { label: 'Status', key: 'status' },
+                  { label: 'Actions', key: 'actions', hideLabel: true },
```

Comment:
Why do we have the labels here and i18n? Either one or the other

Reply:
I forgot to remove them. I’ve done so now.